android/src/main/java/com/reactnativestripesdk/FinancialConnectionsSheetFragment.kt

```diff
@@ -27,6 +27,18 @@ class FinancialConnectionsSheetFragment : Fragment() {
   private lateinit var configuration: FinancialConnectionsSheet.Configuration
   private lateinit var mode: Mode
 
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+
+    val stripeSdkModule: StripeSdkModule? = context.getNativeModule(StripeSdkModule::class.java)
+    if (stripeSdkModule != null && stripeSdkModule.eventListenerCount > 0) {
```

Comment:
same here, do something like this: https://github.com/stripe/stripe-react-native/blob/447804f5529fe91c2f6ffbf983d3151ece283ba4/android/src/main/java/com/reactnativestripesdk/PaymentSheetFragment.kt#L143-L148